tests/code/test_certification.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+from peewee import DoesNotExist
+
+from app.models import mainDB
+from app.models.event import Event
+from app.models.certification import Certification
+from app.models.certificationRequirement import CertificationRequirement
+from app.models.requirementMatch import RequirementMatch
+
+from app.logic.certification import getCertRequirements, updateCertRequirements, updateCertRequirementForEvent
+
+@pytest.mark.integration
+def test_getCertRequirements():
+    allreqs = getCertRequirements()
+
+    certNames =  ["Bonner", "CESC Minor", "CPR", "Confidentiality", "I9"]
+    assert certNames == [cert["data"].name for (id, cert) in allreqs.items()]
+
+    cprReqs = allreqs[3]['requirements']
+    assert ["Volunteer Training", "CPR Training"] == [r.name for r in cprReqs]
+
+    cescreqs = getCertRequirements(certification=Certification.CESC)
+    bonnerreqs = getCertRequirements(certification=Certification.BONNER)
+    assert len(bonnerreqs) == 9
+
+    nonereqs = getCertRequirements(certification=1111)
+    assert len(nonereqs) == 0
+
+@pytest.mark.integration
+def test_updateCertRequirements():
+
+    with mainDB.atomic() as transaction:
+
+        cprcert = 3
+        othercert = 4
```

Comment:
Same here, otherCert would be a better option. Also, I might be confused to understand what otherCert means, is it all the certificates other than the CPR certificate?

Reply:
yes, it is a different certificate id to use for the test. one that is not cpr